src/frontend/main/src/styles/login.css

```diff
@@ -0,0 +1,21 @@
+button {
+    border-radius: 5px;
+    background: #54af29;
```

Comment:
I think styled components and MUI theme are being used for this project:

https://github.com/hotosm/fmtm/blob/main/src/frontend/main/src/utilities/CustomizedMenus.tsx#L7

Reply:
updated for PR